index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <title>My Portfolio</title>
+</head>
+
+<body>
+  <main>
+    <header>
+      <nav class="container">
+        <a href="#">My logo</a>
+        <img class="menu" src="./images/Icon - Menu.png" alt="Menu Icon">
+      </nav>
+      <section>
+        <div class="intro">
+          <div class="first-heading">
+            <h1>I’m Margaret. Glad to see you!</h1>
+          </div>
+          <div class="description">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+              experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          </div>
+    
+        </div>
+        <div >
+          <p class="connect">LET'S CONNECT</p>

```

Comment:
- [ ] kindly use an hx(2 or 3) for the `lets connect` title, as it is a subtitle. This way will be semantically correct, and the screen readers and other technologies will thank you.